lms/djangoapps/instructor/views/tools.py

```diff
@@ -127,13 +128,19 @@ def get_units_with_due_date(course):
     """
     units = []
 
+    # Pass in a schedule here so that we get back any relative dates in the course, but actual value
+    # doesn't matter, since we don't care about the dates themselves, just whether they exist.
+    # Thus we don't save or care about this temporary schedule object.
+    start_schedule = Schedule(start_date=course.start)
```

Comment:
nit: schedule_start instead?  start_schedule sounds like a verb to me idk

Reply:
Hmm, I see what you're saying. I was thinking it as "start" is an adjective, describing the type of schedule we have sitting there. Maybe "temp_schedule" or just "schedule"....

Reply:
Renamed to `schedule`